Sync pre-industrial+emissions

blimlim opened the pull request:
Closes #199 

This PR updates the dev-preindustrial+emissions branch to be up-to-date with the dev-preindustrial+concentrations branch.

[Comparison with the dev-preindustrial+concentrations branch](https://github.com/ACCESS-NRI/access-esm1.6-configs/compare/dev-preindustrial+concentrations..199-update-dev-preindustrial+emissions)

It cherry-picks the following changes from the dev-preindustrial+concentrations branch:
- Using CICE5 https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/147
- Updating the ocean grid https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/171
- Removing the unused calendar settings https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/176
- Updating to the latest executables https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/195

The emissions driven configuration requires `CO2 3D TRACER`, `ATM TRACER 1`, `ATM TRACER 2` to be set to the CMIP7 value of 4.3189e-4 in the atmospheric restart dump. ~I've copied the AugustSpinup initial restart from `/g/data/vk83/prerelease/configurations/inputs/access-esm1p6/modern/pre-industrial/restart/2025.08.01` to `/g/data/vk83/prerelease/configurations/inputs/access-esm1p6/modern/pre-industrial-emissions/restart/2025.09.01` and set the tracer values following the instructions here: https://github.com/ACCESS-NRI/access-esm1.6-configs/issues/141#issue-3161523607~ (see here for details on the latest restart https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/201#issuecomment-3379062305)

blimlim commented:
!test repro commit

blimlim commented:
@dougiesquire, @anton-seaice, @MartinDix – would you be happy to have a look over the changes here?

The comparison with the dev-preindustrial+concentrations branch might be the most relevant thing to check: https://github.com/ACCESS-NRI/access-esm1.6-configs/compare/dev-preindustrial+concentrations..199-update-dev-preindustrial+emissions

dougiesquire reviewed (approved):
Thanks @blimlim, this looks good to me based on the cherry-picked commits. But I'm not across the preindustrial->emissions differences, so probably worth also getting someone else's eyes.

Should there be any changes on the ocean side for the emissions driven config. MOM receives the co2 concentration and returns the co2 flux. Is that all that the UM expects/needs (possibly a question for @MartinDix)?

blimlim commented:
>  I didn't check the "CO2 tracer fields in atmosphere/restart_dump.astart reset to PI control value." has worked because I don't know how!

Good question, I don't know the best way to compare the UM restarts directly either. I've converted them to netCDF ([slightly involved process here](https://github.com/ACCESS-NRI/um2nc-standalone/issues/191#issuecomment-2670080051)), and then compared every variable:

```
for var in concentrations_restart:
    if not concentrations_restart[var].equals(emissions_restart[var]):
        print(var)

>fld_s00i252
>fld_s33i001
>fld_s33i002

-----------
for var in ["fld_s00i252", "fld_s33i001", "fld_s33i002"]:
    print(np.unique(emissions_restart[var].data))

>[0.00043189]
>[0.00043189]
>[0.00043189]
```

blimlim commented:
I've added in the latest updates from the dev-preindustrial+concentrations configuration, cherry picking the commits from:
- OneAPI + openmpi update + Fe file change https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/209
- Reordering the `prefix.PRESMA` file https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/216
- Using the compressed iceberg file: https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/205
- Updating the solar constant and volcanic forcing values: https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/222
- Config changes for the Australian PFTS: https://github.com/ACCESS-NRI/access-esm1.6-configs/pull/194


See [here for a comparison with the dev-preindustrial+concentrations branch](https://github.com/ACCESS-NRI/access-esm1.6-configs/compare/dev-preindustrial+concentrations..199-update-dev-preindustrial+emissions)


For the new restart, I've copied restart230 (`/scratch/p66/jxs599/access-esm/archive/test-PIspinup-Sept-B-test-PIspinup-Sept-B-1b270bbf/restart230`) from the `test-PIspinup-Sept-B` simulation, and added the CO2 values following the [instructions here](https://github.com/ACCESS-NRI/access-esm1.6-configs/issues/141#issue-3161523607).

@MartinDix, this one should be ready for the final review!

blimlim commented:
!test repro commit
